CRM/Civicase/Page/CaseAngular.php

```diff
@@ -16,14 +16,13 @@ class CRM_Civicase_Page_CaseAngular extends \CRM_Core_Page {
    *   The content generated by running this page
    */
   public function run() {
-    $loader = new \Civi\Angular\AngularLoader();
+    $loader = \Civi::service('angularjs.loader');
     $loader
       ->setPageName('civicrm/case/a')
       ->setModules(['civicase'])
```

Comment:
FYI This should be 
```suggestion
      ->addModules(['civicase'])
```
because if multiple angular apps are to coexist peacefully on the same page and use the same loader service, they shouldn't overwrite each other's modules.